index.php

```diff
@@ -1 +1,26 @@
 <?php
+
+if ($_SERVER["REQUEST_METHOD"]=="POST") {
```

Comment:
It's strongly recommended to use $_SERVER['REQUEST'_METHOD'] === 'POST'.
(meaning: type comparison ===)